js/pictures.ts

```diff
@@ -0,0 +1,45 @@
+import { generatePhotosArray } from './mock';
+import { Photo } from './mock';
+
+const picturesElement: HTMLElement | null = document.querySelector('.pictures');
```

Comment:
Можно записывать через Generic
```suggestion
const picturesElement = document.querySelector<HTMLElement>('.pictures');
```